src/routes/collaborator.ts

```diff
@@ -0,0 +1,20 @@
+import { Router } from 'express';
```

Comment:
Let's create a folder for `github` routing as we are going to have a few other paths, including `aws` and `dns`